.github/workflows/pr-operator.yml

```diff
@@ -0,0 +1,400 @@
+  DEFAULT_BUNDLE_CHANNEL: "alpha"
+  DEFAULT_OPERATOR_VERSION: "latest"
+  HELM_REPO_DIR: "./tmp/gh-pages"
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    name: setup
+    steps:
+      - name: Setting Workflow Variables
+        id: set-variables
+        env:
+          BUILD_PLATFORMS: ${{ inputs.BUILD_PLATFORMS }}
+        run: |
+          echo "::set-output name=repository_name::$(basename $GITHUB_REPOSITORY)"
+          echo "::set-output name=bin_dir::$(pwd)/bin"
+
+          # Create Distribution Matrix
+          echo "::set-output name=dist_matrix::$(echo -n "${{ env.BUILD_PLATFORMS }}" | jq -csR '. | split(",")')"
+          # Create Image Tags
+          echo "::set-output name=image_platform_tags::$(echo $BUILD_PLATFORMS | sed  -e 's/,/ /g' -e 's/\//-/g')"
+
+      - name: Setting Image Variables
+        id: set-variables-image
+        run: |
+          if [ "${{ secrets.OPERATOR_IMAGE_REPOSITORY }}" == "" ]; then
+            echo "::set-output name=operator_image_repository_name::${{ steps.set-variables.outputs.repository_name }}"
+            echo "::set-output name=operator_image_registry::quay.io/${{ github.repository_owner }}"
+          else
+            OPERATOR_IMAGE_REPOSITORY="${{ secrets.OPERATOR_IMAGE_REPOSITORY }}"
+            echo "::set-output name=operator_image_repository_name::${OPERATOR_IMAGE_REPOSITORY##*/}"
+            echo "::set-output name=operator_image_registry::${OPERATOR_IMAGE_REPOSITORY%/*}"
+          fi
+
+          if [ "${{ secrets.BUNDLE_IMAGE_REPOSITORY }}" == "" ]; then
+            echo "::set-output name=bundle_image_repository_name::${{ steps.set-variables.outputs.repository_name }}-bundle"
+            echo "::set-output name=bundle_image_registry::quay.io/${{ github.repository_owner }}"
+
+          else
+            BUNDLE_IMAGE_REPOSITORY="${{ secrets.BUNDLE_IMAGE_REPOSITORY }}"
+            echo "::set-output name=bundle_image_repository_name::${BUNDLE_IMAGE_REPOSITORY##*/}"
+            echo "::set-output name=bundle_image_registry::${BUNDLE_IMAGE_REPOSITORY%/*}"
+          fi
+
+          # Set versions based on presence of tag
+          if [[ "${{ github.ref }}" =~ ^refs/tags/ ]]; then
+            TAG="${GITHUB_REF/refs\/tags\//}"
+            echo "::set-output name=tag_event::true"
+            echo "::set-output name=operator_version::$TAG"
+            echo "::set-output name=bundle_version::${TAG:1}"
+          else
+            echo "::set-output name=tag_event::false"
+            echo "::set-output name=operator_version::$DEFAULT_OPERATOR_VERSION"
+            echo "::set-output name=bundle_version::$DEFAULT_BUNDLE_VERSION"
+          fi
+
+      - name: Build Go Cache Paths
+        id: go-cache-paths
+        run: |
+          echo "::set-output name=go-build::$(go env GOCACHE)"
+          echo "::set-output name=go-mod::$(go env GOMODCACHE)"
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v1
+        with:
+          go-version: ^1.16
```

Comment:
can we make this an input variable, with a default instead of a hard-coded value?

Reply:
Added